src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -223,113 +223,140 @@ const NetworkModificationNodeEditor = () => {
+            dialog: null,
+            subItems: {
+                LOAD_CREATION: {
+                    id: 'LOAD_CREATION',
+                    label: 'LOAD',
+                    dialog: () => adapt(LoadCreationDialog),
+                },
+                GENERATOR_CREATION: {
+                    id: 'GENERATOR_CREATION',
+                    label: 'GENERATOR',
+                    dialog: () => adapt(GeneratorCreationDialog),
+                },
+                SHUNT_COMPENSATOR_CREATION: {
+                    id: 'SHUNT_COMPENSATOR_CREATION',
+                    label: 'ShuntCompensator',
+                    dialog: () => adapt(ShuntCompensatorCreationDialog),
+                },
+                LINE_CREATION: {
+                    id: 'LINE_CREATION',
+                    label: 'LINE',
+                    dialog: () => adapt(LineCreationDialog),
+                },
+                TWO_WINDINGS_TRANSFORMER_CREATION: {
+                    id: 'TWO_WINDINGS_TRANSFORMER_CREATION',
+                    label: 'TWO_WINDINGS_TRANSFORMER',
+                    dialog: () => adapt(TwoWindingsTransformerCreationDialog),
+                },
+                VOLTAGE_LEVEL_CREATION: {
+                    id: 'VOLTAGE_LEVEL_CREATION',
+                    label: 'VOLTAGE_LEVEL',
+                    dialog: () => adapt(VoltageLevelCreationDialog),
+                },
+                SUBSTATION_CREATION: {
+                    id: 'SUBSTATION_CREATION',
+                    label: 'SUBSTATION',
+                    dialog: () => adapt(SubstationCreationDialog),
+                },
+            },
         },
-        LOAD_MODIFICATION: {
-            label: 'ModifyLoad',
-            dialog: () => adapt(LoadModificationDialog),
-            icon: <AddIcon />,
+        {
+            id: 'EDIT',
+            label: 'edit',
+            dialog: null,
+            subItems: {
+                LOAD_MODIFICATION: {
+                    label: 'LOAD',
+                    dialog: () => adapt(LoadModificationDialog),
+                },
+                GENERATOR_MODIFICATION: {
+                    label: 'GENERATOR',
+                    dialog: () => adapt(GeneratorModificationDialog),
+                },
+                LINE_MODIFICATION: {
+                    label: 'LINE',
+                    dialog: () => adapt(LineModificationDialog),
+                },
+                VOLTAGE_LEVEL_MODIFICATION: {
+                    label: 'VoltageLevel',
+                    dialog: () => adapt(VoltageLevelModificationDialog),
+                },
+                SUBSTATION_MODIFICATION: {
+                    label: 'SUBSTATION',
+                    dialog: () => adapt(SubstationModificationDialog),
+                },
+            },
         },
-        GENERATOR_CREATION: {
-            label: 'CreateGenerator',
-            dialog: () => adapt(GeneratorCreationDialog),
-            icon: <AddIcon />,
+        {
+            id: 'DELETE',
```

Comment:
You should delete the sub menu here. And 'delete' should be a button you can click to open the delete dialog

Reply:
done

Reply:
done